scripts/update-lib-version.bash

```diff
@@ -19,10 +19,12 @@ echo "Search for files in $(pwd)"
 # lines to update start with something like "<script src="../../demo/0.1.6/"
 if [[ "$OSTYPE" == "darwin"* ]]; then
 	sed -i '' -E "s#<script src=\"../../demo/[^/]+/#<script src=\"../../demo/$NEW_VERSION/#" **/*.{html,md}
+	sed -i '' -E "s/\/demo\/[0-9].[0-9].[0-9]/\/demo\/$NEW_VERSION/#" **/*.js
 else
 	sed -i -E "s#<script src=\"../../demo/[^/]+/#<script src=\"../../demo/$NEW_VERSION/#" **/*.{html,md}
+	sed -i -E "s/\/demo\/[0-9].[0-9].[0-9]/\/demo\/$NEW_VERSION/#" **/*.js
```

Comment:
@aibcmars I have tested this on Ubuntu it works, but
- the `utils.js` is not changed
- we can remove the sed using the `<script src` pattern and keep only the new sed command you added: if we add the html and md pattern, this single sed command update all files

Reply:
fixed with 1fffd4a